### ISSUE TITLE
Invalid Syntax

### ISSUE THREAD

**Issue by patarol** [gugarosa/viviner#1]
Hi @gugarosa 

I'm trying to run your script, however it throws an error when I run `python scrap_wine_data.py -h`:

```python
File "scrap_wine_data.py", line 52
    print(f'Number of matches: {n_matches}')
                                          ^
````

It fails again in:

```python
res = r.get(f'wines/{wine["id"]}/tastes')

# File "scrap_wine_data.py", line 84
#    res = r.get(f'wines/{wine["id"]}/tastes')
```

Any clue how to solve this?
Thank you

**Comment by gugarosa** [comment]
Hello Luciano! I hope everything is going well with you.

It looks like that you are using a Python version smaller than 3.6, which was the initial version that string interpolation (f-string) has been implemented.

Please let me know if that works.

Best regards,
Gustavo.

**Comment by patarol** [comment]
Hi @gugarosa Thank you for your quick response. I managed to te `f-string`, however I'm not sure how to use your code now.
Should we use an URL as input?

After running
```python
python scrap_wine_list.py -h
```
It returns:
```python
#usage: Scraps all wine URLs from Vivino.

#positional arguments:
#  output_file           Output .txt file

#optional arguments:
#  -h, --help            show this help message and exit
#  -start_page START_PAGE
#                        Starting page identifier
```

**Comment by patarol** [comment]
Hi @gugarosa 

I've managed to retrieve the information. It was because I was using `python 2.7` (default in MacOSx). It seems now that the api is blocking the number of requests. 

How did you managed to get all info from Brazilian wines without being blocked?
Thank you again

**Comment by gugarosa** [comment]
If I am not mistaken, I did face this problem when I dumped the dataset and eventually just used a `sleep()` and left the computer running for a few days in order to dump all the information. Nevertheless, it has been a while ago, so they might have changed the number of requests or even the API blocking time.

Something that might give you a head start (if the dumped information matches your needs) is the pre-dumped dataset, which can be found here: https://www.kaggle.com/gthrosa/vivino-brazilian-wines-metadata

Hope that it helps at all!

Best regards,
Gustavo.
